velox/connectors/hive/SplitReader.cpp

```diff
@@ -250,14 +250,20 @@ void SplitReader::setNullConstantValue(
 namespace {
 
 template <TypeKind ToKind>
-velox::variant convertFromString(const std::optional<std::string>& value) {
+velox::variant convertFromString(
+    const std::optional<std::string>& value,
+    const TypePtr& toType) {
   if (value.has_value()) {
     if constexpr (ToKind == TypeKind::VARCHAR) {
       return velox::variant(value.value());
     }
     if constexpr (ToKind == TypeKind::VARBINARY) {
       return velox::variant::binary((value.value()));
     }
+    if (toType->isDate()) {
```

Comment:
Would you add documentation to HiveConnectorSplit::partitionKeys to clarify what values / types are supported and how they are converted from the std::string?

Reply:
Thanks for the suggestion. Updated PR description and documentation.